### ISSUE TITLE
Creating a field with a space leads to the app crashing

### ISSUE THREAD

**Issue by waynehamadi** [twentyhq/twenty#5536]
## Bug Description
Creating a field with a space leads to the app crashing.
https://www.loom.com/share/3f725afdd6ea48f5b3a440677123e52c

Sorry there's going to be a lot of bugs like this. Hopefully I can just stick to shooting loom videos like this, so I don't spend too much time writing the bug.

**Comment by waynehamadi** [comment]
Updates:
https://www.loom.com/share/6b949832419046f782394496bfbfb98d

Currently Twenty is unusable through the web app because of this bug. I can't update the field and make it inactive through REST so I will have to do it through graphql.

**Comment by ijreilly** [comment]
Hi @waynehamadi thanks for the feedback, and sorry about the issue! No object nor field creation should be authorized with a name with spaces. We are fixing that, thank you. 

It seems there is another issue with the request type on https://docs.twenty.com/rest-api/metadata#/operations/updateOneField - also checking on this at the moment, but to unblock you I have deleted the wrongly named field (I saw you created a different one with the right name). It should be good now but please let me know if you have further issues, thank you for your patience and for using the product :)

**Comment by waynehamadi** [comment]
Thanks a lot that was quick
